Sources/Components/Label/Label.swift

```diff
@@ -0,0 +1,147 @@
+import Foundation
+import SwiftUI
+
+extension Warp {
+    /// A customizable label component used to display a title, optional additional information, and an info tooltip.
+    ///
+    /// The `Warp.Label` component displays a main title with optional additional information. It can also show an info icon which, when tapped, presents custom tooltip content provided by the user.
+    ///
+    /// **Usage:**
+    ///
+    /// ```swift
+    /// Warp.Label(
+    ///     title: "Label Title",
+    ///     additionalInformation: "Optional Info",
+    ///     showToolTipImage: true,
```

Comment:
Everywhere else we are using Tootip, but here it is `showToolTipImage` maybe it should be `showTooltipImage`